loom/graphics/gfxTexture.cpp

```diff
@@ -277,6 +282,8 @@ void bitmapExtrudeRGBA_c(const void *srcMip, void *mip, int srcHeight, int srcWi
 
 TextureInfo *Texture::load(uint8_t *data, uint16_t width, uint16_t height, TextureID id)
 {
+	id &= TEXTURE_ID_MASK;
```

Comment:
Don't we have a generate function to generate the IDs? I want to make this pretty bullet proof as it is a fundamental element.